_includes/cloud-query/cloud-query-choose-db.md

```diff
@@ -1,6 +1,6 @@
-There are two methods for defining the database you want to query:
+There are two methods for defining the database you want to query in the UI:
 
 |Method | Instructions  |
 | --- | ----------- |
```

Comment:
I'm pretty sure `| --- |` means the content in the column is centred. Would be worth double-checking?

Reply:
It's left oriented when rendered. Might be a way to have styling to define that 🤷